### PR TITLE
Feat: Added Home-page

### PR DESCRIPTION
**Pull request by SaurabhChawda**
I have added home-page in video-Library.It contain various features:-
- SearchBar user can search video based on the interest.
- Navigation and Sidebar user can switch from one page to another Page.
Screenshot (354)

### CODE REVIEW

File: src/components/Nav/Nav.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+              className="nav-bar__img--logo nav-bar__img--mobile-logo"
+              src="/assets/image/HomePage/Discover-Mobile-Logo.png"
+              alt="Discover"
+            />
+          </a>
+        </nav>
+        <SearchBar />
+        <ul className="nav-bar__list">
+          <li className="nav-bar__item">
+            {/* <img className="nav-bar__img--login" src="/assets/image/HomePage/Login.png" alt="Login" /> */}
```
Comment (ankit9015) [inline src/components/Nav/Nav.jsx:26]: Remove console.log and commented code before merge

File: src/components/Nav/Nav.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            <MdLogin size={30} color="white" />
+          </li>
+          <li className="nav-bar__item">
+            {/* <img className="nav-bar__toggle" src="/assets/image/HomePage/DarkMode.png"></img> */}
```
Comment (ankit9015) [inline src/components/Nav/Nav.jsx:30]: Here also

File: src/components/NavForTab/NavForTab.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import "./NavForTab.css";
+import { MdPlaylistAdd, MdWatchLater, MdOutlineHome, MdOutlineHistoryToggleOff } from "react-icons/md";
+import { AiFillLike } from "react-icons/ai";
+export const NavForTab = () => {
+  return (
+    <div className="nav-bar-tab">
+      <div className="nav-bar-tab-container">
+        <a>
+          {/* <img className="nav-bar-tab--icon" src="/assets/image/HomePage/Home.png" alt="Home" /> */}
```
Comment (ankit9015) [inline src/components/NavForTab/NavForTab.jsx:9]: Here also

File: src/components/Sidebar/Sidebar.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            <MdOutlineHome size={30} color="white" />
+            <h3 className="sidebar--content">Home</h3>
+          </li>
+          <li className="sidebar__list">
+            {/* <img className="sidebar__icon" src="/assets/image/HomePage/Watch Later.png" alt="Watch Later" /> */}
+            <MdWatchLater size={28} color="white" />
+            <h3 className="sidebar--content">Watch Later</h3>
+          </li>
+          <li className="sidebar__list">
+            {/* <img className="sidebar__icon" src="/assets/image/HomePage/Play List.png" alt="Play List" /> */}
```
Comment (ankit9015) [inline src/components/Sidebar/Sidebar.jsx:20]: Here also

File: src/components/Card/Card.css [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  color: var(--white-color);
+  background-color: var(--black-color);
+  border: 2px solid var(--black-color);
+  border-radius: 0.5rem;
+  padding: 2px 6px;
+  position: absolute;
+  right: 10px;
+  bottom: 10px;
+}
+/* .card__dropdown {
```
Comment (ankit9015) [inline src/components/Card/Card.css:63]: Remove commented code

**Review by ankit9015 — APPROVED** [review]
Code is good to merge. 
Try to avoid image files when you can use icons, for performance reasons. Also, you can use external images

File: src/components/Nav/Nav.jsx (hunk 1 above)
Comment (SaurabhChawda) [reply src/components/Nav/Nav.jsx:26]: Okay, i Will make Changes,ThankYou for Review.

**Review by ShreyashMahajan — APPROVED** [review]
good to merge, some suggestions have been mentioned by Ankit, correct them and then good to go

**Comment by SaurabhChawda** [comment]
> good to merge, some suggestions have been mentioned by Ankit, correct them and then good to go

okay,ThankYou

**Pull request merged**